src/api/v1/paths/doughs.js

```diff
@@ -1,38 +1,32 @@
 import { errorHandler } from 'errors/errors';
-import { getPets, postPet } from '../db/json/pets-dao-example';
+import { getDoughs, createDough } from '../db/oracledb/doughs-dao';
 
 /**
- * Get pets
+ * Get doughs
  *
  * @type {RequestHandler}
  */
 const get = async (req, res) => {
   try {
-    const result = await getPets(req.query);
+    const result = await getDoughs(req.query);
     return res.send(result);
   } catch (err) {
     return errorHandler(res, err);
   }
 };
 
-
 /**
- * Post pets
+ * Post doughs
```

Comment:
In theory, _this_ function shouldn't change in the final version. It's definitely not final though since the ticket is just for `GET /doughs`--I just needed to have something here so the server would run.